install-macosx.command

```diff
@@ -7,10 +7,7 @@ brew --version > /dev/null || (echo 'No Homebrew detected. Please install Homebr
 export PATH=/usr/local/bin:$PATH
 set -v
 brew install python
```

Comment:
1. This installs python 3. According to the readme, this is supposed to run under Python 2. macOS ships with 2.7.10 so this could actually be skipped.
2. If you already have the package is installed this will actually fail